config/crds/kudo_v1alpha1_framework.yaml

```diff
@@ -27,6 +27,25 @@ spec:
         metadata:
           type: object
         spec:
+          properties:
+            description:
+              type: string
+            kubeVersion:
```

Comment:
IMO this should be `kubernetesVersion` to avoid any confusion as pointed out here: https://github.com/kudobuilder/kudo/pull/263#discussion_r288985481

Reply:
Agreed. KEP 9 will be adjusted (thanks @hectorj2f). The original motivation for kubeVersion was consistency with Helm's `Chart.yaml` usage of it.